polaris-core/src/main/java/org/apache/polaris/core/storage/StorageAccessProperty.java

```diff
@@ -52,6 +54,21 @@ public enum StorageAccessProperty {
       "the endpoint to load vended credentials for a table from the catalog",
       false,
       false),
+  AWS_REMOTE_SIGNING_ENABLED(
+      Boolean.class,
+      S3FileIOProperties.REMOTE_SIGNING_ENABLED,
+      "whether to enable remote signing for S3 requests",
+      false),
+  AWS_REMOTE_SIGNER_URI(
+      String.class,
+      S3V4RestSignerClient.S3_SIGNER_URI,
+      "the base URI for the remote signer service, used for signing S3 requests",
```

Comment:
Does this have a reference Iceberg version number where the property becomes respected?